source/settings.cpp

```diff
@@ -265,7 +265,8 @@ namespace Settings {
   Option LakeHyliaOwl        = Option::Bool("Lake Hylia Owl",         {"Don't Skip", "Skip"},                                                 {lakeHyliaOwlDesc},                                                                                               OptionCategory::Setting,    SKIP);
   Option BigPoeTargetCount   = Option::U8  ("Big Poe Target Count",   {NumOpts(1, 10)},                                                       {bigPoeTargetCountDesc});
   Option NumRequiredCuccos   = Option::U8  ("Cuccos to return",       {NumOpts(0, 7)},                                                        {numRequiredCuccosDesc});
-  Option KingZoraSpeed       = Option::U8  ("King Zora Speed",        {"Fast", "Vanilla", "Random"},                                          {kingZoraSpeedFast, kingZoraSpeedVanilla, kingZoraSpeedRandom});
+  Option KingZoraSpeed       = Option::U8  ("King Zora Speed",        {"Fast", "Vanilla", "Random", "Custom"},                                {kingZoraSpeedFast, kingZoraSpeedVanilla, kingZoraSpeedRandom, "Manually set how many shuffles King Zora takes."});
+  Option ExactZoraSpeed      = Option::U8  ("  Exact Shuffle Count",  {NumOpts(1,128)},                                                      {""});
```

Comment:
- Descriptions go into `descriptions.cpp`. Add the new one here: https://github.com/gamestabled/OoT3D_Randomizer/blob/c24bca840e46786bc2ae4032a0eb2683fd2bb6f7/source/descriptions.cpp#L613-L621
- Rename the option for clarity, since there's nothing shuffled here
- Make sure the spacing isn't off
```suggestion
  Option KingZoraSpeed       = Option::U8  ("King Zora Speed",        {"Fast", "Vanilla", "Random", "Custom"},                                {kingZoraSpeedFast, kingZoraSpeedVanilla, kingZoraSpeedRandom, kingZoraSpeedCustom});
  Option ExactZoraSpeed      = Option::U8  ("  Exact Mweep Count",    {NumOpts(1,128)},                                                       {""});
```

Reply:
Alright, I saw the whole descriptions.cpp thing I just don't get the point - where I come from we are usually competing with each other for less lines of code while still being legible. I'll go ahead and change that I guess.

Which option? exactZoraSpeed?

It wasn't off in my editor, is there somewhere specifically that I missed?

Reply:
Also, to make those changes do I need to close the Pull Request?

Reply:
> Alright, I saw the whole descriptions.cpp thing I just don't get the point - where I come from we are usually competing with each other for less lines of code while still being legible. I'll go ahead and change that I guess.

Well, just imagine how the settings file would look if all descriptions were in it directly. It's also good for in case someone else eventually wanted to add another option after yours, then they don't have to clean up after you

> Which option? exactZoraSpeed?

Yeah

> It wasn't off in my editor, is there somewhere specifically that I missed?

The little description part was off by one, but it's good now

Reply:
So all good or is there something I missed?

Reply:
The option is still called `Exact Shuffle Count`, or do you think that makes more sense?

Reply:
To me that makes sense, especially since the option is indented and only visible after setting King Zora Speed to Custom. But it's not hard to change, what would you recommend? I can't think of anything else.

Reply:
OH You're talking about changing it to Mweep. Yeah, to be honest I definitely think shuffle count makes more sense. I always hear King Zora saying "Aweek" LOL.